src/spikeinterface/core/baserecording.py

```diff
@@ -321,6 +321,32 @@ def has_scaled_traces(self):
         """
         return self.has_scaled()
 
+    def get_times_kwargs(self, segment_index=None) -> dict:
```

Comment:
```suggestion
    def get_times_info(self, segment_index=None) -> dict:
```

Reply:
This is fine by me, the only thing that I would like is for it to be:

`get_time_info` without the plural.

Why?

We have a function called `get_times` and the `get_times_info` is not -exclusively- about it. So there is a 

I am thinking on something along the lines of:
`get_temporal_info` 

To totally disambiguate from times which are a non explicit property of the recording.

Maybe 
`get_temporal_parameters` 
?

Reply:
yeah you are right, `get_time_info` sounds good to me. Also `temporal` is a good name, but it's a bit to abstract!

Reply:
Yeah, I also don't fully like temporal.

The time information of your recorder.
The temporal information of your recorder.
The "word that condenses how your recorder handles time" information of your recorder.

Couldn't come with anything better.

Reply:
get_time_info is ok for me.

Reply:
Let's do that one then.